test/fixtures/schools.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: "Rice University"
+  school_type: "University"
+  location: "Houston, TX"
+  is_kip: false
+  is_child_of_kip: false
+
+two:
+  name: "PS14"
+  school_type: "High School"
+  location: "New York, New York"
+  is_kip: true
+  is_child_of_kip: false
```

Comment:
Fine to use fixtures as a first step, but as soon as you need something more complex (which should be pretty much right away), use https://github.com/thoughtbot/factory_bot_rails

Reply:
updated to use factory bot - removed fixtures